tests/setup.py

```diff
@@ -0,0 +1,9 @@
+"""
+Setup file for testing purpose.
+
+Session.generate_base_venvs executes `pip ... install -e .` and
```

Comment:
Why was this not needed before?

Reply:
with #183 merged I think we don't need this anymore